(window.webpackJsonp=window.webpackJsonp||[]).push([[0],{23:function(e,t,a){e.exports=a(53)},28:function(e,t,a){},43:function(e,t,a){},47:function(e,t,a){},48:function(e,t,a){},49:function(e,t,a){},50:function(e,t,a){},51:function(e,t,a){},52:function(e,t,a){},53:function(e,t,a){"use strict";a.r(t);var n=a(0),i=a.n(n),o=a(14),l=a.n(o),r=(a(28),a(15)),s=a(16),c=a(21),m=a(17),p=a(22),u=a(5),d=a(2),h={title:"NM_024675.3(PALB2):c.298C>T (p.Leu100Phe)",tooltip:"For ClinVar alleles, this represents the ClinVar Preferred Title. For alleles not in ClinVar, this HGVS is based on the transcript with the longest translation with no stop codons or, if no translation, the longest non-protein-coding transcript. If a single canonical transcript is not discernible the HGVS is based on the GRCh38 genomic coordinates.",subtitle:"This interpretation is not yet associated with a disease or mode of inheritance"},g={sourcesTitle:"Variant ID Sources",sources:[{name:"ClinVar VaraitionID",link:"126698",url:"https://www.ncbi.nlm.nih.gov/clinvar/variation/126698"},{name:"dbSNP ID",link:"rs61756147",url:"https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61756147"}],genomicTitle:"Variant Genomic Context",genomicContext:[{name:"UCSC",links:[{url:"https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&amp;position=chr16%3A23636218-23636278",copy:"GRCh48/hg38",title:"UCSC Genome Browser for NC_000016.10:g.23636248G>A in a new window"},{url:"https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr16%3A23647539-23647599",copy:"GRCh37/hg19",title:"UCSC Genome Browser for NC_000016.9:g.23647569G>A in a new window"}]},{name:"Variation Viewer",links:[{url:"https://www.ncbi.nlm.nih.gov/variation/view/?chr=16&amp;assm=GCF_000001405.28&amp;from=23636218&amp;to=23636278",copy:"GRCh48",title:"Variation Viewer page for NC_000016.10:g.23636248G>A in a new window"},{url:"https://www.ncbi.nlm.nih.gov/variation/view/?chr=16&assm=GCF_000001405.25&from=23647539&to=23647599",copy:"GRCh37",title:"Variation Viewer page for NC_000016.9:g.23647569G>A in a new window"}]},{name:"Ensembl Browser",links:[{url:"http://uswest.ensembl.org/Homo_sapiens/Location/View?db=core;r=16:23636218-23636278",copy:"GRCh48",title:"Ensembl Browser page for NC_000016.10:g.23636248G>A in a new window"},{url:"http://grch47.ensembl.org/Homo_sapiens/Location/View?db=core;r=16:23647539-23647599",copy:"GRCh47",title:"Ensembl Browser page for NC_000016.9:g.23647569G>A in a new window"}]}],interpretations:{title:"My Interpretation",content:[{title:"Disease",value:"Not provided"},{title:"Calculated Pathogenicity",value:"None"},{title:"Modified Pathogenicity",value:"None"},{title:"Provisional/Approved Status",value:""}]}},b=[{class:"stand-alone",copy:"BA1",tooltip:"Allele frequency greater than 5% in a population database"},{class:"stand-alone",copy:"BS1",tooltip:"MAF is too high for disorder"},{class:"stand-alone",copy:"BS2",tooltip:"Observation in controls inconsistent with disease penetrance"},{class:"stand-alone",copy:"BS3",tooltip:"Well-established functional studies show no deleterious effect"},{class:"stand-alone",copy:"BS4",tooltip:"Non-segregation with disease"},{class:"benign-supporting",copy:"BP1",tooltip:"Missense in gene where primarily truncating cause disease"},{class:"benign-supporting",copy:"BP2",tooltip:"Observed in trans with path. variant for dominant disorder or cis any inheritance"},{class:"benign-supporting",copy:"BP3",tooltip:"In-frame indels in repeat w/out known function"},{class:"benign-supporting",copy:"BP4",tooltip:"Multiple lines of computational evidence suggest no impact on gene /gene product"},{class:"benign-supporting",copy:"BP5",tooltip:"Found in case with an alternate cause"},{class:"benign-supporting",copy:"BP6",tooltip:"Reputable source w/out shared data = benign"},{class:"benign-supporting",copy:"BP7",tooltip:"Silent variant predicted with no splice impact"},{class:"pathogenic-supporting",copy:"PP1",tooltip:"Cosegregation with disease in multiple affected family members"},{class:"pathogenic-supporting",copy:"PP2",tooltip:"Missense in gene with low rate of benign missense variants and path. missenses common"},{class:"pathogenic-supporting",copy:"PP3",tooltip:"Multiple lines of computational evidence support a deleterious effect on the gene /gene product"},{class:"pathogenic-supporting",copy:"PP4",tooltip:"Patient's phenotype or FH highly specific for gene"},{class:"pathogenic-supporting",copy:"PP5",tooltip:"Reputable source = pathogenic"},{class:"pathogenic-moderate",copy:"PM1",tooltip:"Mutational hot spot or well-studied functional domain without benign variation"},{class:"pathogenic-moderate",copy:"PM2",tooltip:"Absent in population databases"},{class:"pathogenic-moderate",copy:"PM3",tooltip:"For recessive disorders, detected in trans with a pathogenic variant"},{class:"pathogenic-moderate",copy:"PM4",tooltip:"Protein length changing variant"},{class:"pathogenic-moderate",copy:"PM5",tooltip:"Novel missense change at an amino acid residue where a different pathogenic missense change has been seen before"},{class:"pathogenic-moderate",copy:"PM6",tooltip:"De novo (without paternity & maternity confirmed)"},{class:"pathogenic-strong",copy:"PS1",tooltip:"Same amino acid change as an established pathogenic variant"},{class:"pathogenic-strong",copy:"PS2",tooltip:"De novo (paternity and maternity confirmed)"},{class:"pathogenic-strong",copy:"PS3",tooltip:"Well-established functional studies show a deleterious effect"},{class:"pathogenic-strong",copy:"PS4",tooltip:"Prevalence in affecteds statistically increased over controls"},{class:"pathogenic-very-strong",copy:"PSV1",tooltip:"Predicted null variant in a gene where LOF is a known mechanism of disease"}],E={title:"NM_024675.3(PALB2):c.298C>T (p.Leu100Phe)",tooltip:"For ClinVar alleles, this represents the ClinVar Preferred Title. For alleles not in ClinVar, this HGVS is based on the transcript with the longest translation with no stop codons or, if no translation, the longest non-protein-coding transcript. If a single canonical transcript is not discernible the HGVS is based on the GRCh38 genomic coordinates.",subtitle:"This interpretation is not yet associated with a disease or mode of inheritance"},f=[{iconColor:"#62387e",title:"Benign",copy:"No criteria met"},{iconColor:"#d33535",title:"Pathogenic",copy:"No criteria met"},{iconColor:"#666666",title:"Calculated Pathogenicity",copy:"None"}],y=a(9),v=a.n(y),w=a(4),N=a(3),C=(a(43),a(13),function(){return i.a.createElement(S,null,function(e){var t=e.variantTitleBox;return i.a.createElement("div",{className:"title-box"},i.a.createElement("button",null,"View Summary"),i.a.createElement("h1",null,t.title,i.a.createElement(v.a,{action:"hover",placement:"top",arrowFill:"rgba(0, 0, 0, 0.8)"},i.a.createElement("span",null,i.a.createElement(w.a,{icon:N.d})),i.a.createElement("div",{className:"ra-tooltip"},t.tooltip))),i.a.createElement("h2",null,t.subtitle))})}),T=a(18),P=a.n(T);a(47);function F(){return i.a.createElement(S,null,function(e){var t=e.curationDataContext;return i.a.createElement(P.a,{width:"32%",gap:"1%",className:"grid-container"},i.a.createElement("div",{className:"grid-item greenBg"},i.a.createElement("h4",null,t.sourcesTitle),i.a.createElement("dl",null,t.sources.map(function(e,t){return i.a.createElement("dd",{key:t},e.name,":\xa0",i.a.createElement("a",{href:e.url,target:"_blank",rel:"noopener noreferrer"},e.link,"\xa0",i.a.createElement(w.a,{icon:N.c})))}))),i.a.createElement("div",{className:"grid-item greenBg"},i.a.createElement("h4",null,t.genomicTitle),i.a.createElement("dl",null,t.genomicContext.map(function(e,t){return i.a.createElement("dd",{key:t},e.name,": [",e.links.map(function(e,t){return i.a.createElement("span",{className:"link",key:t},i.a.createElement("a",{href:e.url,title:e.title,target:"_blank",rel:"noopener noreferrer"},e.copy,"\xa0",i.a.createElement(w.a,{icon:N.c})))}),"]")}))),i.a.createElement("div",{className:"grid-item greenBg"},i.a.createElement("h4",null,t.interpretations.title),i.a.createElement("dl",null,t.interpretations.content.map(function(e,t){return i.a.createElement("dd",{key:t},i.a.createElement("strong",null,e.title,":")," ",e.value)}))))})}a(48);function B(){return i.a.createElement(S,null,function(e){var t=e.curationCriteria;return i.a.createElement("div",{className:"curation-criteria"},t.map(function(e,t){return i.a.createElement("div",{key:t,className:"btn"},i.a.createElement(v.a,{action:"hover",placement:"bottom",arrowFill:"rgba(0, 0, 0, 0.8)"},i.a.createElement("button",{className:e.class},e.copy),i.a.createElement("div",{className:"ra-tooltip"},e.tooltip)))}))})}a(49);function I(){return i.a.createElement(S,null,function(e){var t=e.progressBarCriteria;return i.a.createElement("div",{className:"flex-container criteriaBox"},t.map(function(e,t){var a;return a="#666666"===e.iconColor?N.a:N.b,i.a.createElement("div",{className:"flex-col",key:t},i.a.createElement("div",{className:"criteriaIcon"},i.a.createElement(w.a,{icon:a,color:e.iconColor})),i.a.createElement("div",{className:"criteriaCopy"},e.title,i.a.createElement("br",null),i.a.createElement("span",{className:"criteriaValue"},e.copy)))}))})}a(50);function G(){return i.a.createElement(S,null,function(e){e.basicinfoCriteria;return i.a.createElement(i.a.Fragment,null,i.a.createElement("div",{className:"callout genomic"},i.a.createElement("h4",null,"Genomic"),i.a.createElement("ul",null,i.a.createElement("li",null,i.a.createElement("span",{className:"title-ellipsis title-ellipsis-short"},"NC_000016.10:g.23636248G>A"),i.a.createElement("span",null," (GRCh38)")),i.a.createElement("li",null,i.a.createElement("span",{className:"title-ellipsis title-ellipsis-short"},"NC_000016.9:g.23647569G>A"),i.a.createElement("span",null," (GRCh37)")))),i.a.createElement("div",{className:"panel panel-info all-existing-interpretaions"},i.a.createElement("div",{className:"panel-heading"},i.a.createElement("h3",{className:"panel-title"},"All interpretations for this variant in the Variant Curation Interface (VCI)",i.a.createElement("span",{className:"interpretation-status-explanation"},i.a.createElement("span",{className:"text-info contextual-help","data-toggle":"tooltip","data-placement":"top","data-tooltip":'Interpretations marked as "Approved" may be viewed by any user within the interface; those marked as "In progress" or "Provisional" are viewable only by the submitter.'},i.a.createElement("i",{className:"icon icon-info-circle"}))))),i.a.createElement("div",{className:"panel-content-wrapper"},i.a.createElement("div",{className:"all-existing-interpretaions-content-wrapper"},i.a.createElement("table",{className:"table"},i.a.createElement("thead",null,i.a.createElement("tr",null,i.a.createElement("th",null,"Clinical significance"),i.a.createElement("th",null,"Status"),i.a.createElement("th",null,"Condition - ",i.a.createElement("i",null,"Mode of inheritance")),i.a.createElement("th",null,"Curator/Affiliation"))),i.a.createElement("tbody",null,i.a.createElement("tr",{className:"approved-interpretation"},i.a.createElement("td",{className:"clinical-significance"},"--"),i.a.createElement("td",{className:"interpretation-status"},i.a.createElement("span",{className:"no-classification"},"None")),i.a.createElement("td",{className:"condition-mode-of-inheritance"},i.a.createElement("span",null,"Not provided")),i.a.createElement("td",{className:"submitter"},i.a.createElement("a",{href:"mailto:clingen.demo.curator@genome.stanford.edu"},"ClinGen Test Curator")))))))))})}a(51),a(52);var k=i.a.createContext(),x=function(e){function t(e){var a;return Object(r.a)(this,t),(a=Object(c.a)(this,Object(m.a)(t).call(this,e))).componentDidMount=function(){a.getData(Object(u.a)(Object(u.a)(a)))},a.getData=function(e){var t=e;fetch("https://mygene.info/v3/query?q=79728&species=human&fields=entrezgene,exac,HGNC,MIM,homologene.id,interpro,name,pathway.kegg,pathway.netpath,pathway.pid,pdb,pfam,pharmgkb,prosite,uniprot.Swiss-Prot,summary,symbol").then(function(e){if(e.status>=400)throw new Error("Bad response from server");return e.json()}).then(function(e){console.log(e),t.setState({variant:e})})},a.state={variant:!1},a.componentDidMount=a.componentDidMount.bind(Object(u.a)(Object(u.a)(a))),a}return Object(p.a)(t,e),Object(s.a)(t,[{key:"render",value:function(){return i.a.createElement(k.Provider,{value:{variantTitleBox:h,curationDataContext:g,curationCriteria:b,basicinfoCriteria:E,progressBarCriteria:f,tabs:["Basic Information"]}},i.a.createElement("div",{className:"App"},i.a.createElement(C,null),i.a.createElement(F,null),i.a.createElement(B,null),i.a.createElement(I,null),i.a.createElement(d.Wrapper,{letterNavigation:!0},i.a.createElement(d.TabList,null,i.a.createElement("ul",{className:"FancyTabs-tablist"},i.a.createElement("li",{className:"FancyTabs-tablistItem"},i.a.createElement(d.Tab,{id:"t1",className:"FancyTabs-tab"},V.bind(null,i.a.createElement("span",{className:"FancyTabs-tabText"},"Basic Information")))),i.a.createElement("li",{className:"FancyTabs-tablistItem"},i.a.createElement(d.Tab,{id:"t2",className:"FancyTabs-tab"},V.bind(null,i.a.createElement("span",{className:"FancyTabs-tabText"},"Population")))),i.a.createElement("li",{className:"FancyTabs-tablistItem"},i.a.createElement(d.Tab,{id:"t3",className:"FancyTabs-tab"},V.bind(null,i.a.createElement("span",{className:"FancyTabs-tabText"},"Variant Type")))),i.a.createElement("li",{className:"FancyTabs-tablistItem"},i.a.createElement(d.Tab,{id:"t4",className:"FancyTabs-tab"},V.bind(null,i.a.createElement("span",{className:"FancyTabs-tabText"},"Case/Segregation")))),i.a.createElement("li",{className:"FancyTabs-tablistItem"},i.a.createElement(d.Tab,{id:"t5",className:"FancyTabs-tab"},V.bind(null,i.a.createElement("span",{className:"FancyTabs-tabText"},"Gene-centric")))))),i.a.createElement("div",{className:"FancyTabs-panel"},i.a.createElement(d.TabPanel,{tabId:"t1"},i.a.createElement("div",{className:"FancyTabs-panelInner"},i.a.createElement(G,null))),i.a.createElement(d.TabPanel,{tabId:"t2"},i.a.createElement("div",{className:"FancyTabs-panelInner"},"Ut ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.")),i.a.createElement(d.TabPanel,{tabId:"t3"},i.a.createElement("div",{className:"FancyTabs-panelInner"},"Duis irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.")),i.a.createElement(d.TabPanel,{tabId:"t4"},i.a.createElement("div",{className:"FancyTabs-panelInner"},"Duis irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.")),i.a.createElement(d.TabPanel,{tabId:"t5"},i.a.createElement("div",{className:"FancyTabs-panelInner"},"Duis irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."))))))}}]),t}(n.Component);function V(e,t){var a="FancyTabs-tabInner";return t.isActive&&(a+=" is-active"),i.a.createElement("div",{className:a},e)}var S=k.Consumer;Boolean("localhost"===window.location.hostname||"[::1]"===window.location.hostname||window.location.hostname.match(/^127(?:\.(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$/));l.a.render(i.a.createElement(x,null),document.getElementById("root")),"serviceWorker"in navigator&&navigator.serviceWorker.ready.then(function(e){e.unregister()})}},[[23,1,2]]]);
//# sourceMappingURL=main.980a2a63.chunk.js.map